servers/rendering/shader_preprocessor.cpp

```diff
@@ -543,7 +543,7 @@ void ShaderPreprocessor::process_else(Tokenizer *p_tokenizer) {
 	}
 	state->current_branch->else_defined = true;
 
-	if (state->save_regions) {
+	if (state->save_regions && state->previous_region) {
```

Comment:
What is this from?

Reply:
Accidental commit of unrelated changes I was looking into submitting later on.